### DISCUSSION TITLE
Drafts - How to show More Drafts

### DISCUSSION THREAD

**Discussion by dawghouse73** [nmelhado/league-page#223] [Q&A]
I'm not sure if this is a bug or an enhancement or maybe both. 

League ID: 922637695908737024
Site: shulacesoutdan.vercel.app

Currently, by default, our Drafts page shows two items:
1. Next Season Draft Board (2024)
2. Rookie Draft (held in May 2023)

What's missing: 
- Our Initial Veterans Only Start Up draft from February 2023
- The Draft Board for 2025

We allow the trading of picks for the next rookie draft and the following rookie draft. It would be nice to see the board and the picks owned by each manager for both next year and the year after. What I'm hoping to be able to see is the following:
![image](https://github.com/nmelhado/league-page/assets/133917715/a9921290-418e-4bfa-b88e-f70695cb21e6)

**Comment by dawghouse73** [comment]
Just checking in on this ...
Even if you can't pull the very first draft data in (although, the API should be able)...
can you point me to how I can place a static image on the bottom of the Drafts page so I can include the image snippet above ?
I'm not learned in the SVELTE ways.

Thanks
![Inaugural Draft Board - February 2023](https://github.com/nmelhado/league-page/assets/133917715/78816d85-e81f-4d5a-85f2-2286f1f8ee47)

**Reply by nmelhado** [reply]
Sorry, completely forgot about this. Do you mind opening up a bug issue for the missing draft and separately a feature request for the future drafts?

I try to monitor all of the discussions, but I use the issues tab as my guide whenever I'm doing work.

**Comment by dawghouse73** [comment]
Will do. -----------Jason HouseOn Jul 29, 2023, at 4:01 AM, Nicholas Melhado ***@***.***> wrote:﻿
Sorry, completely forgot about this. Do you mind opening up a bug issue for the missing draft and separately a feature request for the future drafts?
I try to monitor all of the discussions, but I use the issues tab as my guide whenever I'm doing work.

**Reply by dawghouse73** [reply]
Entered: #240
